hydra/_internal/hydra.py

```diff
@@ -207,6 +210,8 @@ def format_args_help(args_parser: ArgumentParser) -> str:
             if len(action.option_strings) == 0:
                 overrides = action
             else:
+                if action.help == SUPPRESS:
+                    continue
```

Comment:
what's this?

Reply:
The `--_completion` flag should be hidden from the user. [`SUPPRESS`](https://docs.python.org/3/library/argparse.html#help) is a flag to silence options in argparse.